AdfsEventsModule.psm1

```diff
@@ -546,7 +1204,8 @@ function Write-ADFSEventsSummary
     $table.columns.add( $col6 )
     $table.columns.add( $col7 )
 
-    foreach($Event in $input.Events){
+    foreach($Event in $input.Events)

```

Comment:
Instead of using $input, I suggest adding a parameter and then using [AllowInputFromPipeline] attribute on that parameter. $input works differently in different scenarios. Here's someone's blog: https://dmitrysotnikov.wordpress.com/2008/11/26/input-gotchas/

Reply:
Thanks Ali, fixed as per your suggestion